logux-vue-todomvc/src/server/configure.js

```diff
@@ -0,0 +1,85 @@
+const { Server } = require('@logux/server')
+const { nanoid } = require('nanoid')
+
+const server = new Server(
+  Server.loadOptions(process, {
+    controlSecret: 'secret',
+    subprotocol: '1.0.0',
+    supports: '1.0.0',
+  })
+)
+
+server.auth((userId, token) => {
+  return true
```

Comment:
We need a good example of authentification in the example, since it will be an example how to use Logux.

Reply:
do you have any idea how it should look like? window with login at start and separate todos for each user?

Reply:
Sign up page with login and signup forms on the same page. We can use Logux to send email/password. The server will return a JWT token.

Something like https://logux.io/recipes/authentication/#method-2:-everything-in-logux

Reply:
I end up with complicated code (too complicated for such simple goal).

And `changeUser` works slow due to cross tab leader election. May be it's better not to reconnect web socket or at least add special case to leader election?